repository-data/application/src/main/resources/hcm-config/namespaces/hee/report.yaml

```diff
@@ -56,37 +48,65 @@ definitions:
             hipposysedit:path: hee:pageLastNextReview
             hipposysedit:primary: false
             hipposysedit:type: hee:pageLastNextReview
+            hipposysedit:validators: [required]
           /rightHandBlocks:
             jcr:primaryType: hipposysedit:field
             hipposysedit:multiple: true
             hipposysedit:ordered: true
             hipposysedit:path: hee:rightHandBlocks
             hipposysedit:type: hippo:compound
             hipposysedit:validators: [contentblocks-validator]
-          /logoGroup:
+          /string:
             jcr:primaryType: hipposysedit:field
             hipposysedit:mandatory: false
             hipposysedit:multiple: false
             hipposysedit:ordered: false
-            hipposysedit:path: hee:logoGroup
+            hipposysedit:path: hee:subtitle
             hipposysedit:primary: false
-            hipposysedit:type: hippo:mirror
+            hipposysedit:type: String
             hipposysedit:validators: [optional]
-          /microHero:
+          /publicationDate:
             jcr:primaryType: hipposysedit:field
             hipposysedit:mandatory: false
             hipposysedit:multiple: false
             hipposysedit:ordered: false
-            hipposysedit:path: hee:microHero
+            hipposysedit:path: hee:publicationDate
             hipposysedit:primary: false
-            hipposysedit:type: hippogallerypicker:imagelink
-            hipposysedit:validators: [optional]
+            hipposysedit:type: CalendarDate
+            hipposysedit:validators: [required]
+          /publicationType:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:publicationType
+            hipposysedit:primary: false
+            hipposysedit:type: DynamicDropdown
+            hipposysedit:validators: [required]
+          /publicationtopics:
```

Comment:
Could we camelCase this [`publicationTopics`] (including `hipposysedit:path` property) to be inline with other field names ?